### PR TITLE
Bug fix - Loader on Standings screen

### PR DESCRIPTION
**Pull request by ksookdhew**
## Description
Fixed loader error with standings

## Checklist 
Please review and check the following before submitting this pull request: 
- [x] The code follows the project's coding standards. 
- [x] 100% Unit test coverage for View Models 
- [x] All warnings Resolved.

### CODE REVIEW

**Review by kiasha10 — DISMISSED** [review]

File: F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift [hunk 1]
```diff
@@ -35,15 +33,16 @@ class ConstructorStandingsViewModel {
         return firstTwoCodes.joined(separator: "/")
     }
 
-    func fetchConstructorStandings() {
-        isLoaded = true
+    func fetchConstructorStandings(completion: @escaping ([ConstructorStanding]?) -> Void) {
```
Comment (kieran-woodrow) [inline F1/F1/Standings/ConstructorStandings/ConstructorStandingsViewModel.swift:36]: why does this need a completion now?

File: F1/F1Tests/StandingsViewModelTests.swift [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+////  Created by Kaitlyn Sookdhew on 2024/05/05.
+////
 //
-//  StandingsViewModelTests.swift
-//  F1Tests
+//import XCTest
+//@testable import F1
 //
-//  Created by Kaitlyn Sookdhew on 2024/05/05.
+//class MockStandingsNavigationDelegate: StandingsNavigationDelegate {
```
Comment (kieran-woodrow) [inline F1/F1Tests/StandingsViewModelTests.swift:11]: Please don't commit empty/commented out files to dev

File: F1/F1/Standings/StandigsViewModel.swift [hunk 3]
```diff
@@ -35,4 +36,41 @@ class StandingsViewModel {
             }
         }
     }
+
+    func isLoaded() -> Bool {
```
Comment (kieran-woodrow) [inline F1/F1/Standings/StandigsViewModel.swift:40]: can be a computed property rather

File: F1/F1/Standings/StandigsViewModel.swift [hunk 4]
```diff
@@ -35,4 +36,41 @@ class StandingsViewModel {
...
+        driverViewModel.isLoaded && constructorViewModel.isLoaded
+    }
+
+    func fetchStandings() {
+        let dispatchGroup = DispatchGroup()
```
Comment (kieran-woodrow) [inline F1/F1/Standings/StandigsViewModel.swift:45]: why do you need to use concurrency. If it is for loading, there are better ways to handle that with delegates and UIloaders

File: F1/F1Tests/StandingsViewModelTests.swift (hunk 2 above)
Comment (ksookdhew) [reply F1/F1Tests/StandingsViewModelTests.swift:11]: Sorry 😄

File: F1/F1/Standings/StandigsViewModel.swift (hunk 4 above)
Comment (kieran-woodrow) [reply F1/F1/Standings/StandigsViewModel.swift:45]: chatted offline

**Review by kiasha10 — APPROVED** [review]

**Review by simekanidvt — APPROVED** [review]

**Pull request merged**